packages/react-cookie/src/tests/cookie.ts

```diff
@@ -0,0 +1,15 @@
+export default class Cookie {
+  mock(value: string) {
```

Comment:
should this go live in `jest-dom-mocks`?

Reply:
yeah it should 👍

Reply:
I'm going to do this in a follow up though. There are already a lot of packages involved in this PR and this is an easy thing to extract afterward.